scripts/gbmake

```diff
@@ -307,7 +307,7 @@ gnubee_bin() {
 
 gnubee_firmware() {
     { [ -f "$GNUBEE_KERNEL_OBJECTS/.config" ] || gnubee_defconfig ${1}; } &&
-    make O="$GNUBEE_KERNEL_OBJECTS" ${2--j4} &&
+    make O="$GNUBEE_KERNEL_OBJECTS" ${2--j$(nproc)} &&
```

Comment:
This is effectively the same as a bare "-j" - which removes any limit and lets the scheduler make use of whatever is available.

I find that letting "make "use all processors can cause other workloads on the same machine to suffer, so I'm not very keen on this as a default choice.  Maybe add a setting for the config file which allows the "-j4" to be over-ridden.  e.g. pass "-j4 $GNUBEE_MAKE_ARGS $2" to make here.  Later "-j" flags over-ride earlier ones so setting "GNUBEE_MAKE_ARGS=-j$(nproc)" in your "config" will do what you want.

Either way I'd prefer this as a separate pull request.

Thanks,

Reply:
> This is effectively the same as a bare "-j" - which removes any limit and lets the scheduler make use of whatever is available.

I didn't even know that! Thanks for mentioning this.

> 
> I find that letting "make "use all processors can cause other workloads on the same machine to suffer, so I'm not very keen on this as a default choice. Maybe add a setting for the config file which allows the "-j4" to be over-ridden. e.g. pass "-j4 GNUBEEMAKEARGS(nproc)" in your "config" will do what you want.

This makes sense, this patch can go away.